src/App.tsx

```diff
@@ -1,18 +1,35 @@
-import React from 'react';
+/* eslint-disable no-console */
+import React, { useState } from 'react';
 
 import './App.scss';
+import { Tabs } from './Tabs';
 
-const tabs: Tab[] = [
+export const tabs: Tab[] = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Selected tab is&nbsp;Tab 1</h1>
-    <div>{tabs.length}</div>
-  </div>
-);
+const App: React.FC = () => {
+  const [selectedTab, setSelectedTab] = useState(tabs[0]);
+
+  const onTabSelected = (tab: Tab) => {
+    setSelectedTab(tab);
+  };
+
+  return (
+    <div className="App">
+      <h1>Selected tab is&nbsp;Tab 1</h1>
+      Selected Tab is
+      {' '}
+      {selectedTab.title}
+      <Tabs
+        tabs={tabs}
+        selectedTabId={selectedTab.id}
```

Comment:
```suggestion
        selectedTab={selectedTab}
```
По условию хоть и надо было передать айди, чтоб потом найти селектнутый таб, но какой в этом смысл если можно сразу передать весь таб

Reply:
передаю id, тому що потім по id шукаю табу, яка потрібна :)